src/world/common/enemy/ai/PatrolNoAttackAI.inc.c

```diff
@@ -1,3 +1,6 @@
+#ifndef _PATROL_AI_NO_ATTACK_
```

Comment:
Is there a specific reason for adding an include guard to an inc.c file?

Reply:
Several of them need these. Multiple enemy AIs include this, and when those enemies are included in the same maps together, only the first included PatrolNoAttackAI/etc appears on the final ROM. Based on comparisons between different maps, we can work out which parts of the larger includes are common and where guards are needed with fairly high confidence that the original sources were organized along similar lines.

Reply:
I shee. okay sounds good